<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreVoirNoteEtudiant</class>
 <widget class="QWidget" name="FenetreVoirNoteEtudiant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FenetreVoirNoteEtudiant</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#boutonVoirNote {
	font: 75 14pt &quot;Tw Cen MT&quot;;
	border: 1px solid #ccc;
	border-radius: 4px;
	background-color: rgb(55, 94, 173);
	color: #fff;
}

QPushButton#boutonVoirNote:hover {
	background-color: rgb(47, 80, 147);
}

QMessageBox { 
	font-size: 15px;
 }

labelResultat{
	background-color: #fff;
}

QLabel {
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
	font-weight: bold;
	text-align: center;
}

QComboBox {
     border: 1px solid gray;
     border-radius: 3px;
     padding: 1px 18px 1px 3px;
     min-width: 6em;
 }
 
 QComboBox:editable {
     background: white;
 }
 
 QComboBox:!editable, QComboBox::drop-down:editable {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                  stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                  stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
 }

 QComboBox:!editable:on, QComboBox::drop-down:editable:on {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                 stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
 }
 
 QComboBox:on {
     padding-top: 3px;
     padding-left: 4px;
 }
 
 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;
 
     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid;
     border-top-right-radius: 3px;
     border-bottom-right-radius: 3px;
 }

 QComboBox::down-arrow {
     image: url(:/imgs/images/flecheBas.png);
	 width: 10px;
 }

 QComboBox QAbstractItemView {
     border: 2px solid darkgray;
     selection-background-color: #394da7;
 }

QLabel#labelResultat {
	background-color: #fff;
	border: 1px solid #ccc;
	border-radius: 2px;
}

QScrollArea{	
	background-color: #fff;
	border: 1px solid #ccc;
	border-radius: 2px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacerGauche">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="labelMatriculeEtudiant">
                <property name="text">
                 <string>Matricule de l'Etudiant :</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxMatriculeEtudiant">
                <property name="minimumSize">
                 <size>
                  <width>101</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="editable">
                 <bool>true</bool>
                </property>
                <property name="insertPolicy">
                 <enum>QComboBox::InsertAlphabetically</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="labelNomMatiere">
                <property name="text">
                 <string>Nom de la matière :</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxNomMatiere">
                <property name="minimumSize">
                 <size>
                  <width>101</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="editable">
                 <bool>true</bool>
                </property>
                <property name="insertPolicy">
                 <enum>QComboBox::InsertAlphabetically</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="labelAnneeAcademique">
                <property name="text">
                 <string>Annee Académique :</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxAnneeAcademique">
                <property name="minimumSize">
                 <size>
                  <width>101</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="editable">
                 <bool>true</bool>
                </property>
                <property name="insertPolicy">
                 <enum>QComboBox::InsertAlphabetically</enum>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacerGroupBox">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacerGroupBoxGauche">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>200</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="boutonVoirNote">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>173</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Voir la note</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacerGroupBoxDroit">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>200</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerDroit">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>952</width>
        <height>174</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="labelMatricule">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="labelPrenomEtudiant">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelNomEtudiant">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="labelNoteEtudiant">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="labelMatiere">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QLabel" name="labelAnnee">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
